test/integration/vm-integration-run

```diff
@@ -63,6 +63,12 @@ if [[ -d "${HOME}/src" && -f "${HOME}/src/version" ]]; then
   mv "${HOME}/src" "${HOME}/.kpatch/"
 fi
 
+source /etc/os-release
+
+if [[ "${NAME}" == "Fedora" ]] && [[ "${VERSION_ID}" -le 27 ]]; then
```

Comment:
Why <= 27?  I think CONFIG_LIVEPATCH is disabled for all versions of Fedora?

Reply:
Ah, I didn't realize that. This is not entirely true though, starting with fc30 we do have it enabled: https://src.fedoraproject.org/rpms/kernel/c/3792ab811f3d097b7883cf43235c75ea410b3ebc

I'll fix this up.

Reply:
Ah, nice... I'm glad Fedora enabled it, that will make it easier to test with new kernels.

Eventually we may want to have a rolling "latest Fedora" set of integration tests to catch regressions with new kernels.